src/runtime_src/core/common/api/xrt_module.cpp

```diff
@@ -137,10 +137,13 @@ struct patcher
   buf_type m_buf_type = buf_type::ctrltext;
   symbol_type m_symbol_type = symbol_type::shim_dma_48;
 
+  static constexpr size_t num_bd_data_ptrs = 8; // max number of bd ptrs accessed while patching
   struct patch_info {
```

Comment:
Can we add a comment on why **8** and make this a file constant?

Reply:
Made changes accordingly Sonal, Thanks